src/pages/workspace/download/index.tsx

```diff
@@ -0,0 +1,10 @@
+import type DownloadQRCodeProps from './types';
+
+// eslint-disable-next-line @typescript-eslint/no-unused-vars
+function DownloadMenuItem({download}: DownloadQRCodeProps) {
+    return null;
```

Comment:
Why are we not rendering the download button on web/desktop?

Reply:
We do not do it for all other QR codes - for Share user, Share report page

Reply:
As @shawnborton liked above, it seems like we're refactoring those pages to include the button - https://expensify.slack.com/archives/C036QM0SLJK/p1708435456279989?thread_ts=1708354311.864439&cid=C036QM0SLJK